code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -113,26 +113,6 @@
 
 /datum/species/ethereal/spec_life(mob/living/carbon/human/H)
 	.=..()
-	handle_charge(H)
```

Comment:
You can remove the spec_life instead because handle_charge is called by handle_digestion, I just didn't notice that I left this when I changed how charge works.

Reply:
Acknowledged